DogeNet.IdentityServer/Data/DBInitializer.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DogeNet.IdentityServer.Data
+{
+    public static class DBInitializer
```

Comment:
Empty class?

Reply:
Sorry, forgot about it XD
I'lldelete it